src/main/java/redis/clients/jedis/params/GeoRadiusParam.java

```diff
@@ -18,8 +17,7 @@ public class GeoRadiusParam implements IParams {
 
   private Integer count = null;
   private boolean any = false;
-  private boolean asc = false;
```

Comment:
Would it be better to use an Enum for this case, so we don't get into a case where `asc` and `desc` have conflicting values?

Reply:
@bsbodden That's what I did. These are deleted lines. Please check added line 20.